spec/features/logging/view_auth_requests_for_a_username_spec.rb

```diff
@@ -54,9 +54,15 @@
     context "when username is correct" do
       let(:search_string) { "AAAAAA" }
 
-      it "displays the authentication requests" do
+      it "displays 2 results" do
```

Comment:
pedantic, but anything under ten should be written as a word rather than a number. (some schools of thought say under one hundred)